kore/src/main/java/org/kframework/utils/StringUtil.java

```diff
@@ -822,4 +699,22 @@ public static String decodeKoreString(String encoded) {
         }
         return encoded;
     }
+
+    public static String[] splitOneDimensionalAtt(String att) {
+        String[] splitted = att.trim().split(",");
+        for (int i = 0; i < splitted.length; i++) {
+            splitted[i] = splitted[i].trim();
+        }
+        return splitted;
+    }
+
+    public static String[][] splitTwoDimensionalAtt(String att) {
```

Comment:
I know we went over this, but if you can look again and see if this is re-usable in the places where we are doing this type of splitting, that would be good. I'm going to approve anyway.